src/ffmpeg.imageio/ffmpeginput.cpp

```diff
@@ -206,29 +213,31 @@ FFmpegInput::open (const std::string &name, ImageSpec &spec)
         !strcmp (m_codec_context->codec->name, "dvvideo")) {
         m_offset_time = false;
     }
+
+   m_codec_cap_delay=m_codec_context->codec->capabilities&AV_CODEC_CAP_DELAY ;
```

Comment:
In the version of ffmpeg on one of my systems, AV_CODEC_CAP_DELAY is not found.

But there is a CODEC_CAP_DELAY. Are they the same, and I'm using a slightly older version, after which the AV_ was prepended?

do you think it's sufficient if I just add something like:

    #ifndef AV_CODEC_CAP_DELAY
    #define AV_CODEC_CAP_DELAY CODEC_CAP_DELAY
    #endif